go.mod

```diff
@@ -3,106 +3,110 @@ module github.com/cheqd/did-resolver
 go 1.17
 
 require (
-	github.com/cheqd/cheqd-node v0.6.0
+	github.com/cheqd/cheqd-node v1.0.0-develop.10
```

Comment:
Reminder - change to v1.0.0 once released.

Reply:
Change it to more recent dev release